modules/nextflow/src/main/groovy/nextflow/data/cid/CidStore.groovy

```diff
@@ -39,6 +39,6 @@ interface CidStore {
 
     Path getPath()
 
-    CidHistoryFile getHistoryFile()
+    CidHistoryLog getHistoryLog()
```

Comment:
Considering that's the most important interface, let's add some javadoc to describe it and document methods, params and semantic